packages/Webkul/Admin/src/Http/Controllers/Settings/RoleController.php

```diff
@@ -56,6 +56,7 @@ public function store()
             'name'            => 'required',
             'permission_type' => 'required',
             'description'     => 'required',
+            'permissions'     => 'required_unless:permission_type,all',
```

Comment:
We can use only the required condition here because we need to ensure that at least one role permission is selected.